java/test/jmri/swing/NamedBeanComboBoxTest.java

```diff
@@ -0,0 +1,364 @@
+package jmri.swing;
+
+import java.awt.GraphicsEnvironment;
+import java.awt.SystemColor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.HashSet;
+
+import javax.swing.JList;
+import javax.swing.JLabel;
+import javax.swing.JTextField;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Assume;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import jmri.InstanceManager;
+import jmri.Manager;
+import jmri.Sensor;
+import jmri.SensorManager;
+import jmri.util.JUnitUtil;
+
+/**
+ *
+ * @author Bob Jacobsen Copyright (C) 2017
+ */
+public class NamedBeanComboBoxTest {
+
+    @Test
+    public void testSensorSimpleCtor() {
+        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
+        Manager<Sensor> m = InstanceManager.getDefault(jmri.SensorManager.class);
+        NamedBeanComboBox<Sensor> t = new NamedBeanComboBox<>(m);
+        Assert.assertNotNull("exists", t);
```

Comment:
This line is not necessary since a constructor will never return null. You can change the lines
```
NamedBeanComboBox<Sensor> t = new NamedBeanComboBox<>(m);
Assert.assertNotNull("exists", t);
```
to
`new NamedBeanComboBox<>(m);`

Reply:
I'd rather have redundant code in the test than get into the practice of creating "throw away" objects that are immediately marked for garbage collection, or worse, creating "throw away" objects to achieve a side effect of construction of an object.

Reply:
@danielb987 we do this a lot in tests actually, just to verify something strange did not occur. 

One guideline for test code is that every test should make at least one assertion.  We don’t always follow that rule, but we really should.  ( the only time I make an exception is when testing exceptions. You can specify that the test should throw an exception )

So, the bottom line is that the test is correct as written by @rhwood.